mgmtsystem_survey/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE survey_survey s
+        SET state = 'management_system',
+        FROM survey_stage ss
+        WHERE s.stage_id = ss.id
```

Comment:
Don't use this flaw connection. Instead get the `res_id` of the XML-ID:
```suggestion
            LEFT JOIN ir_model_data imd ON imd.res_id = s.stage_id AND imd.module = 'mgmt_survey' AND imd.name = 'mgmtsystem_survey_type'
```